### ISSUE TITLE
child_concurrency configuration not used

### ISSUE THREAD

**Issue by victornoel** [yarnpkg/yarn#6118]
There is a child_concurrency configuration (see https://yarnpkg.com/en/docs/envvars#child_concurrency- and https://yarnpkg.com/en/docs/yarnrc#child-concurrency-) but it is apparently not taken into account.

See https://github.com/yarnpkg/yarn/blob/2f0eb8eace4c19c5e4606176371782c60bc7e0e7/src/util/child.js where the constant is used instead of the configuration.

I'm having problems either with #1874 or #3728 and this is preventing me from working around it I think…

**Comment by victornoel** [comment]
@arcanis and will this be solved in 2019? ;)

**Comment by quitedensepoint** [comment]
Yeah, this is pretty annoying. I run into problems with node-gyp via yarn every few days because of https://github.com/yarnpkg/yarn/issues/1874 and can't work around it with CHILD_CONCURRENCY because it doesn't do anything per this issue.
